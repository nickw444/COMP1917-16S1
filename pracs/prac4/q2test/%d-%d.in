9
-1182 -346 992 804 443 45 74 523 117 
