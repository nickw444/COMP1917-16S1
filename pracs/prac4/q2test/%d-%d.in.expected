sum of array: 1470
